### PR TITLE
Infos page display details

### PR DESCRIPTION
**Pull request by vincentGuegan**
modification display email, tel and address

### CODE REVIEW

File: src/main/java/com/quinzeminutespourmoi/quinzePourMoi/config/MySecurityConfig.java [hunk 1]
```diff
@@ -33,6 +33,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/resultSurvey").authenticated()
                 .antMatchers("/hypnoRegister").authenticated()
                 .antMatchers("/users/**").authenticated()
+                .antMatchers("/notification").authenticated()
```
Comment (wasthishelpful) [inline src/main/java/com/quinzeminutespourmoi/quinzePourMoi/config/MySecurityConfig.java]: /notifications ?

**Comment by wasthishelpful** [comment]
conflits à résoudre :/

File: src/main/java/com/quinzeminutespourmoi/quinzePourMoi/config/MySecurityConfig.java (hunk 1 above)
Comment (vincentGuegan) [reply src/main/java/com/quinzeminutespourmoi/quinzePourMoi/config/MySecurityConfig.java]: done

**Comment by vincentGuegan** [comment]
conflicts corrected

**Pull request merged**